src/modules/auth/components/EnterPasswordForm/enterPasswordForm.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import WalletVisual from '@wallet/components/walletVisual';
+import useDecryptionAccount from '@account/hooks/useDecryptionAccount';
+import { Input } from 'src/theme';
+import Box from 'src/theme/box';
+import BoxContent from 'src/theme/box/content';
+import { PrimaryButton } from 'src/theme/buttons';
+import styles from './enterPasswordForm.css';
+
+const EnterPasswordForm = ({ accountSchema, onEnterPasswordSuccess }) => {
+  const { t } = useTranslation();
+  const [password, setPassword] = useState('');
+  const [feedbackError, setFeedbackError] = useState('');
+
+  const onSubmit = () => {
+    const { privateToken, recoveryPhrase, error } = useDecryptionAccount(accountSchema, password);
+    if (error) {
+      return setFeedbackError(error);
+    }
+
+    return onEnterPasswordSuccess({ privateToken, recoveryPhrase });
+  };
+
+  return (
+    <Box className={styles.container}>
+      <BoxContent className={styles.content}>
+        <h1>{t('Enter your password')}</h1>
+        <p className={styles.subheader}>
+          {t('Please provide your device password to backup the recovery phrase.')}
+        </p>
+        <WalletVisual
+          className={styles.avatar}
+          address={accountSchema?.metadata?.address}
+        />
+        <p className={styles.accountName}>{accountSchema?.metadata?.name}</p>
+        <p className={styles.accountAddress}>{accountSchema?.metadata?.address}</p>
+        <Input
+          placeholder={t('Enter password')}
+          type="password"
+          value={password}
+          onChange={(e) => {
+            setPassword(e.target.value);
+          }}
+          feedback={feedbackError}
+        />
+        <PrimaryButton
+          className={styles.button}
+          onClick={onSubmit}
+          disabled={!password}
+        >
+          {t('Continue')}
+        </PrimaryButton>
+      </BoxContent>
+    </Box>
```

Comment:
I think this should be managed by a form so that the perks of being able to hit enter and an onsubmit callback is triggered can be preserved.

Also as a suggestion, you can use useForm from `react-hook-form` to manage the form.

Reply:
@eniolam1000752  is just a input without validation, we can keep it like that for now